text/0000-unsafe-pointer-reform.md

```diff
@@ -92,27 +96,34 @@ In conclusion: `as isize` sucks and isn't helpful; let's just get some unsigned
 Add the following method equivalents for the static `ptr` functions on `*const T` and `*mut T`:
 
 ```rust
-ptr.copy(dst: *mut T, count: usize)
-ptr.copy_nonoverlapping(dst: *mut T, count: usize)
-ptr.read() -> T
-ptr.read_volatile() -> T
-ptr.read_unaligned() -> T
+impl<T> *(const|mut) T {
+  unsafe fn read(self) -> T;
+  unsafe fn read_volatile(self) -> T;
+  unsafe fn read_unaligned(self) -> T;
+
+  // NOTE: name changed from the original static fn to make it 
+  // easier to remember argument order
+  unsafe fn copy_to(self, dest: *mut T, count: usize);
+  unsafe fn copy_to_nonoverlapping(self, src: *mut T, count: usize);
+}
 ```
 
 And these only on `*mut T`:
 
 ```rust
-ptr.drop_in_place()
-ptr.write(val: T)
-ptr.write_bytes(val: u8, count: usize)
-ptr.write_volatile(val: T)
-ptr.write_unaligned()
-ptr.replace(val: T) -> T
+impl<T> *mut T {
+  // note that I've moved these from both to just `*mut T`, to go along with `copy_from`
+  unsafe fn drop_in_place(self);
+  unsafe fn write(self, val: T);
+  unsafe fn write_bytes(self, val: u8, count: usize);
+  unsafe fn write_volatile(self, val: T);
+  unsafe fn write_unaligned(self, val: T);
+  unsafe fn replace(self, val: T) -> T;
+  unsafe fn swap(self, with: *mut T);
+}
 ```
 
-`ptr.swap` has been excluded from this proposal because it's a symmetric operation, and is consequently a bit weird to methodize.
-
-The static functions should remain undeprecated, as they are more ergonomic in the cases explained in the motivation.
+Note that this proposal doesn't deprecate the static functions, as they still make some code more ergonomic than methods, and we'd like to avoid regressing the ergonomics of any usecase. (More discussion can be found in the alternatives.)
```

Comment:
Minor point: can this note be moved to the very top of the "Design" section? That way, it is clear that this applies to both the 
"Methodization" and "Unsigned offsets" subsections.